real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -276,11 +303,11 @@ int main(void) {
 
     if(g_is_autonomous){
       steering_set(auto_steering_angle);
-      g_rbsm_endpoint.Send(RBSM_MID_MEGA_STEER_ANGLE, (long int)(auto_steering_angle));
+      g_rbsm.Send(RBSM_MID_MEGA_STEER_ANGLE, (long int)(auto_steering_angle));
     }
     else if(!g_is_autonomous){
       steering_set(steer_angle);
-      g_rbsm_endpoint.Send(RBSM_MID_MEGA_STEER_ANGLE, (long int)steer_angle);
+      g_rbsm.Send(RBSM_MID_MEGA_STEER_ANGLE, (long int)steer_angle);
     }
     else{
       // dbg_println("Somehow not in either autonomous or teleop");
```

Comment:
Can you still fix it, I am trying to get the quality of code on master up a little. 

I think this is a good place for a comment ie //Somehow not in either autonomous or teleop